tests/unit/test_expression_tree/test_unary_operators.py

```diff
@@ -153,19 +165,24 @@ def test_sign(self):
 
         assert pybamm.sign(c) == pybamm.Sign._from_json(sign_json)
 
-    def test_floor(self, mocker):
+    @settings(suppress_health_check=[HealthCheck.function_scoped_fixture])
+    @given(
+        random_value1=st.floats(min_value=-1e3, max_value=1e3),
+        random_value2=st.floats(min_value=-1e3, max_value=1e3),
+    )
```

Comment:
```suggestion
        random_value1=st.floats(min_value=-1e38, max_value=1e38),
        random_value2=st.floats(min_value=-1e38, max_value=1e38),
    )
```
If there's a particular reason you are keeping for small range here and everywhere else, I'll be glad to know.